linux_os/guide/services/mail/postfix_harden_os/postfix_server_cfg/postfix_server_relay/postfix_prevent_unrestricted_relay/oval/shared.xml

```diff
@@ -0,0 +1,27 @@
+<def-group>
+  <definition class="compliance" id="postfix_prevent_unrestricted_relay" version="1">
+    <metadata>
+      <title>Configure Postfix Against Unrestricted Mail Relay</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>Red Hat Enterprise Linux 8</platform>
+		<platform>multi_platform_wrlinux</platform>
```

Comment:
```suggestion
        <platform>multi_platform_wrlinux</platform>
```

Reply:
Yes, not sure where the spacing issue came from. My original file does not look like this. May be a moot point though if I need to change the file to the macro format.